File: back/src/main/java/com/skytrix/model/dto/card/CardSetDTO.java
package com.skytrix.model.dto.card;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class CardSetDTO {
    private Long id;
    private String name;
    private String code;
    private String rarity;
    private String rarityCode;
    private float price;
    private long cardId;
}

File: front/src/app/core/model/dto/card-set-dto.ts
export class CardSetDTO {
  id: number;
  name?: string;
  code?: string;
  rarity?: string;
  rarityCode?: string;
  price?: number;
  cardId?: number;

  constructor(
    id: number,
    name?: string,
    code?: string,
    rarity?: string,
    rarityCode?: string,
    price?: number,
    cardId?: number
  ) {
    this.id = id;
    this.name = name;
    this.code = code;
    this.rarity = rarity;
    this.rarityCode = rarityCode;
    this.price = price;
    this.cardId = cardId;
  }
}
